src/modules/system.cc

```diff
@@ -58,6 +58,16 @@ NAN_METHOD(GetSize) {
     RETURN(Nan::New<Number>((uint32_t) GetObjectSize(info[0].As<Object>())));
 }
 
+NAN_METHOD(ConvertGValue) {
+    Local<Object> obj = info[0].As<Object>();
+    if (!ValueHasInternalField(obj)) {
+        RETURN(Nan::Undefined());
+        return;
+    }
+    void *ptr = obj->GetAlignedPointerFromInternalField (0);
+    RETURN(GValueToV8(reinterpret_cast<GValue *>(ptr), false));
```

Comment:
Can you add a variable `mustCopy`? It's more readable when you run into it than simply a boolean flag.
Also it will need to be `true` here.